README.md

```diff
@@ -841,23 +841,26 @@ Kafka top-level configuration contains generic options that fit both publisher a
 | Cert PEM Key             | Client's private key string (PEM format) used for authentication  | `cert_pem_key`             | `APP_PUBSUB_KAFKA_CERT_PEM_KEY`             | string       | long PEM key string (deprecated)                      |
 | Security Protocol        | Protocol used to communicate with brokers                         | `security_protocol`        | `APP_PUBSUB_KAFKA_SECURITY_PROTOCOL`        | string       | plaintext, ssl, sasl_plaintext, sasl_ssl (deprecated) |
 | SSL verification enabled | Enable OpenSSL's builtin broker (server) certificate verification | `ssl_verification_enabled` | `APP_PUBSUB_KAFKA_SSL_VERIFICATION_ENABLED` | bool         | true, false (deprecated)                              |
+| Enable Metrics           | Enable publishing metrics for Kafka client broker connection      | `metrics_enabled`          | `APP_PUBSUB_KAFKA_METRICS_ENABLED`          | bool         | true, false                                           |
```

Comment:
Nit suggestion:k

```suggestion
| Enable Metrics           | Enable publishing metrics for Kafka client broker connection      | `metrics_enabled`          | `APP_PUBSUB_KAFKA_BROKER_METRICS_ENABLED`          | bool         | true, false                                           |
```

Maybe adding `BROKER` makes it more clear what this setting is about? 🤔

Reply:
@fotos We don't have `broker` related configuration section atm, and I don't think we need it. We will adjust it in the future if we need one.

Reply:
Sounds good.